raster/r.mapcalc/Makefile

```
@@ -20,7 +20,11 @@ default: multi
 $(BIN)/$(PGM2)$(EXE): LIBES = $(LIBES2)
 $(BIN)/$(PGM3)$(EXE): LIBES = $(LIBES3)
 
-$(OBJDIR)/mapcalc.yy.o: mapcalc.tab.h
+$(OBJDIR)/*.o: mapcalc.tab.META
+$(OBJDIR)/mapcalc.yy.o: mapcalc.tab.META
+
+mapcalc.tab.META: mapcalc.tab.c mapcalc.tab.h
+	touch mapcalc.tab.META
```

```suggestion
$(OBJDIR)/*.o: $(OBJDIR)/mapcalc.tab.META
$(OBJDIR)/mapcalc.yy.o: $(OBJDIR)/mapcalc.tab.META

$(OBJDIR)/mapcalc.tab.META: mapcalc.tab.c mapcalc.tab.h
	touch $(OBJDIR)/mapcalc.tab.META
```

The improvement looks good (thanks!) but may I suggest to put the `.META` file into `$(OBJDIR)`?

Like this we keep the source directory free of temporary files while the `$(OBJDIR)` content is cleaned up by `make clean` and `make distclean`.

---

Great suggestion. I applied it.